modules/apps/batch-engine/batch-engine-test/src/testFunctional/macros/BatchEngine.macro

```diff
@@ -46,21 +46,64 @@ definition {
 			value1 = "move-file -Dfile=${sourceFilePath} -Dtofile=${deployFolder}/${fileName}");
 
 		WaitForConsoleTextPresent(value1 = "Processing ${fileName}");
-
-		// Need to add pause here to finish deploying zip through batch engine LPS-161983
-
-		Pause(locator1 = 5000);
 	}
 
 	macro setUpGlobalIds {
 		var companyId = JSONCompany.getCompanyIdNoSelenium(portalURL = ${portalURL});
 		var siteId = JSONGroupAPI._getSiteIdByGroupKey(groupName = "Guest");
+		var userId = JSONUserAPI._getUserIdByEmailAddress(userEmailAddress = "test@liferay.com");
 		static var staticCompanyId = ${companyId};
 		static var staticSiteId = ${siteId};
+		static var staticUserId = ${userId};
 
```

Comment:
it is added here because JSONT test needs it, I should have done it separately, but forgot to move it :/

Reply:
good to know :+1: